pkg/providers/kube/multicluster_test.go

```diff
@@ -83,4 +83,18 @@ var _ = Describe("Test Multicluster functions of the Kubernetes endpoint", func(
 			Expect(actual).To(Equal(expectedEndpoint))
 		})
 	})
+
+	Context("Test getIPPort()", func() {
```

Comment:
Could we please use go testing, we have decided to deprecate ginko for unit tests

Reply:
I felt that it would be time-efficient to follow the existing pattern.  What I remember is that we agreed to steer towards Go tests, but not opposed to maintaining Ginkgo where they already exist.

Reply:
For new tests, I strongly suggest using the Go testing framework for consistency. There has been an effort individually to rewrite Ginkgo based tests using Go testing, so I suggest rewriting the test here to not rely on Ginkgo.

Reply:
Converted the tests.